README.md

```diff
@@ -54,11 +54,13 @@ For example, for [`channels/fast-4.2.yaml`](channels/fast-4.2.yaml):
 feeder:
   name: candidate-4.2
   delay: P1W
+  errata: public
   filter: 4\.[0-9]+\.[0-9]+(.*hotfix.*|\+amd64|-s390x)?
 ```
 
-which declares the intention that nodes and edges will be considered for promotion into `fast-4.2` after cooking for one week in `candidate-4.2`.
+which declares the intention that nodes and edges will be considered for promotion into `fast-4.2` after cooking for one week in `candidate-4.2` or the errata becomes public.
```

Comment:
iiuc we can't release to fast unless errata is public - should we remove "one week" clause here?

Reply:
I feel like we'll want to go Cincy-first if for some reason the errata sticks, and within-a-week-of-candidate seemed like a reasonable cap on how long we'd wait before wanting to push an errata.  But yeah, I'm fine revisiting with ART to figure out what we want to do in the "has not been tombstoned, but _still_ no errata after $REALLY_LONG_DURATION" case.

Reply:
I've pushed 8a762a6 -> 70c99de dropping `delay` from the fast channels' feeders, so now we'll block forever (but still send Slack notifications) when promotion is blocked on the lack of a public errata.